### ISSUE TITLE
Camera crashes when Beauty Mode set to 0

### ISSUE THREAD

**Issue by XEonAX** [XEonAX/ANXCamera10#2]
Workaround Set to greater than 0

**Comment by amog787** [comment]
@XEonAX ^

**Comment by Akashsystematix** [comment]
Ye but it makes photos less sharp then miui stock

**Comment by RavengamerSpace** [comment]
Yeah, the app says that now he can't connect to your front camera
![Screenshot_20190527-231854_AnX_App _photo](https://user-images.githubusercontent.com/28545669/58439790-5d688b80-80d6-11e9-8c0b-f30145eadfa3.png)

**Comment by opensourcecompany** [comment]
Same problem here.
Even after severan restarts no connection to the camera :-/
